src/common/helpEmailAddress.ts

```diff
@@ -0,0 +1 @@
+export const helpEmailAddress = "help@foodbankapp.org";
```

Comment:
Is there another place where we have a list of constants like this? If not might be worth generalising this file and calling it something like `contactDetails` or `staticInformation` (I'm sure you can come up with better depending on what we want to use this for)

Reply:
I can't find anywhere else with constants like this, so I've renamed it to `contactDetails` for now!